Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -908,11 +908,13 @@ public bool SwitchCurrentControlScheme(params InputDevice[] devices)
                 throw new InvalidOperationException(
                     "Must set actions on PlayerInput in order to be able to switch control schemes");
 
+            // Find control scheme matching given devices in associated action asset
             var scheme = InputControlScheme.FindControlSchemeForDevices(devices, actions.controlSchemes);
-            if (scheme == null)
+            if (!scheme.HasValue)
```

Comment:
I do not believe so since FindControlSchemeForDevices returns a nullable struct wrapping struct InputControlScheme, e.g. https://docs.microsoft.com/en-us/dotnet/api/system.nullable-1?view=net-5.0 struct. Hence scheme?.HasValue would not be valid code since value type. I believe Nullable<T> using comparison operator overload == is equal to HasValue but is there a code convention I am not aware of that we prefer using the operator overload instead?

Reply:
Yep, my bad. Didn't notice it was nullable